### PR TITLE
Version 3.0.9

### PR DESCRIPTION
**Pull request by janhybs**

### CODE REVIEW

**Comment by janhybs** [comment]
linked to the issue #1042

File: bin/fterm [hunk 1]
```diff
@@ -371,6 +371,11 @@ while [[ $# -gt 0 ]]
       usage
       exit 1
     ;;
+    -i|--images)
```
Comment (jbrezmorf) [inline bin/fterm]: Please, add to the help.

File: bin/www/flow123d [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```
Comment (jbrezmorf) [inline bin/www/flow123d]: What is in dir bin/www?

**Comment by jbrezmorf** [comment]
Just a bit of clarification and we can merge this.

**Pull request merged**